.github/workflows/stale.yaml

```diff
@@ -0,0 +1,27 @@
+name: "Close stale issues"
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: >
+          This issue has been automatically marked as stale because it
+          has not had recent activity. It will be closed if no further
+          activity occurs in the next 30 days.
+        stale-pr-message: >
+          This pull request has been automatically marked as stale because it
+          has not had recent activity. It will be closed if no further
+          activity occurs in the next 30 days.
+        days-before-stale: 90
+        days-before-close: 30
+        stale-issue-label: 'status:stale'
```

Comment:
Why `status:stale` instead of `stale`?

Reply:
Because I copied that from another repository :laughing: